Documents/UseCasesForWebsite.md

```diff
@@ -0,0 +1,39 @@
+Actors: Visitor, Artist, Admin, System 
+
+**Visitor** - View info page, View Gallery, Donate, RequestProstetic, Like a Picture.
+A visitor would use this website for getting information or donate as amputee a prosthetic or donate as a company. 
```

Comment:
All use case name should be bolded. 
Every actor should have a general description, not just a list of use cases.